build/pipelines/release.yml

```diff
@@ -265,6 +269,231 @@ jobs:
       IndexSources: false
       SymbolServerType: TeamServices
 
+- ${{ if eq(parameters.runCompliance, true) }}:
+  - job: Compliance
+    # We don't *need* a matrix but there's no other way to set parameters on a "job"
+    # in the AzDO YAML syntax. It would have to be a "stage" or a "template".
+    # Doesn't matter. We're going to do compliance on Release x64 because
+    # that's the one all the tooling works against for sure.
+    strategy:
+      matrix:
+        Release_x64:
+          BuildConfiguration: Release
+          BuildPlatform: x64
+    displayName: Validate Security and Compliance
+    timeoutInMinutes: 240
+    steps:
+    - checkout: self
+      clean: true
+      submodules: true
+      persistCredentials: True
+    - task: PkgESSetupBuild@12
+      displayName: Package ES - Setup Build
+      inputs:
+        disableOutputRedirect: true
+    - task: PowerShell@2
+      displayName: Rationalize Build Platform
+      inputs:
+        targetType: inline
+        script: >-
+          $Arch = "$(BuildPlatform)"
+
+          If ($Arch -Eq "x86") { $Arch = "Win32" }
+
+          Write-Host "##vso[task.setvariable variable=RationalizedBuildPlatform]${Arch}"
+    - task: NuGetToolInstaller@1
+      displayName: Use NuGet 5.10
+      inputs:
+        versionSpec: 5.10
+    - task: NuGetAuthenticate@0
+    # In the Microsoft Azure DevOps tenant, NuGetCommand is ambiguous.
+    # This should be `task: NuGetCommand@2`
+    - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+      displayName: Restore NuGet packages for extraneous build actions
+      inputs:
+        command: restore
+        feedsToUse: config
+        configPath: NuGet.config
+        restoreSolution: build/packages.config
+        restoreDirectory: '$(Build.SourcesDirectory)\packages'
+    - task: NuGetCommand@2
+      displayName: NuGet custom
+      inputs:
+        command: custom
+        selectOrConfig: config
+        nugetConfigPath: NuGet.Config
+        arguments: restore OpenConsole.sln -SolutionDirectory $(Build.SourcesDirectory)
+    - task: UniversalPackages@0
+      displayName: Download terminal-internal Universal Package
+      inputs:
+        feedListDownload: 2b3f8893-a6e8-411f-b197-a9e05576da48
+        packageListDownload: e82d490c-af86-4733-9dc4-07b772033204
+        versionListDownload: $(TerminalInternalPackageVersion)
+    - task: TouchdownBuildTask@1
+      displayName: Download Localization Files
+      inputs:
+        teamId: 7105

```

Comment:
Should we move this to be a variable in the pipeline instead?

Reply:
Maybe. It won't really change and I'm going to look into deduplicating this copy with the other ones in a future YAML revision.